node/src/sub_lib/cryptde.rs

```diff
@@ -377,7 +377,7 @@ impl<'a> Visitor<'a> for CryptDataVisitor {
 
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct PlainData {
-    data: Vec<u8>,
+    pub(crate) data: Vec<u8>,
```

Comment:
please remove that `(crate)` part, it's irritating 


I'm also not much happy that it must be public now, does it?